virttest/utils_time.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import re
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+def get_host_timezone(params):
+    """
+    Timedatectl is installed with systemd.
+    :param params: Dictionary with the test parameters
+
+    :return: timez_city: the host's timezone in city format, used in Linux os.
+    :return: timez_code: the hosts timezone in utc code format, used in both
+                         Linux and Win.
+    """
+    try:
+        o = process.system_output(params["check_host_tz_cmd"], timeout=240,
+                                  shell=True)
+    except process.CmdError:
+        logging.warn("Can not get host's timezone")
```

Comment:
with `timez_city = ""`
`timez_code = ""`declared before `try`, you can, `return` them when there is a CmdError.
Because you would not able to get the `tz` in `o`

Reply:
How about return False here? Cause you can't execute below code without the output.